services/app-web/src/testHelpers/apolloMocks/contractGQLMock.ts

```diff
@@ -62,16 +62,8 @@ const updateDraftContractRatesMockSuccess = ({
                         actuarialFirmOther: ""
                     }
                 ],
-                addtlActuaryContacts: [
-                    {
-                        name: "",
-                        titleRole: "",
-                        email: "",
-                        actuarialFirm: undefined,
-                        actuarialFirmOther: ""
-                    }
-                ],
-                actuaryCommunicationPreference: undefined,
+                addtlActuaryContacts: [],
+                actuaryCommunicationPreference: 'OACT_TO_ACTUARY',
```

Comment:
I'm still looking into this issue. Trying to track down where these values where coming from because they were leading to a mocked value not expected error that.

Reply:
There have been some changes made on main that change how these two fields are populated. Check out these two PRs for more info about the changes

https://github.com/Enterprise-CMCS/managed-care-review/pull/2405
https://github.com/Enterprise-CMCS/managed-care-review/pull/2365/files#diff-ff90de2aafd5581b18aa6d7c34ac384743737b240120b2ad40c37366c4e07658

As for where they come from, they are straight from the `RateRevisionTable` and are no longer manipulated into a structure for the old UI. 

The old way was a workaround for the UI, where we coped one actuary comm preference to all rates and stored it in the rate level `actuaryCommunicationPreference`. For additional actuaries, we did the same, where we coped the same additional actuaries to all rates.

Then for the sake of getting the PRs merged, I just set `addtlActuaryContacts` and actuaryCommunicationPreference at the package/contract level to an empty array or "".

Reply:
Thanks Jason, with merged pulled in things look good